### PR TITLE
Lps 141492

### PR DESCRIPTION
**Pull request by shuyangzhou**

### CODE REVIEW

File: modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java [hunk 1]
```diff
@@ -119,12 +119,7 @@ public void open() {
 				new NamedThreadFactory(
 					getName(), Thread.NORM_PRIORITY,
 					PortalClassLoaderUtil.getClassLoader()),
-				(r, executor) -> {
-					_rejectedTaskCounter.incrementAndGet();
-
-					_rejectedExecutionHandler.rejectedExecution(r, executor);
-				},
-				new ThreadPoolHandlerAdapter());
```
Comment (rotty3000) [inline modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java]: doesn't this lose counting when a rejection handler is provided?

Comment (shuyangzhou) [reply modules/apps/portal/portal-messaging/src/main/java/com/liferay/portal/messaging/internal/BaseAsyncDestination.java]: You are right! Sorry, forget user can set a different one, I will add a commit to keep the wrapping logic when user does set one.

**Comment by shuyangzhou** [comment]
ci:stop

**Comment by shuyangzhou** [comment]
@rotty3000 see https://github.com/shuyangzhou/liferay-portal/pull/10385/commits/9d0f144fce9773a4f1f1a0181ed79a15cd4fb5d2 thanks for catching it!

**Comment by shuyangzhou** [comment]
ci:forward

**Comment by shuyangzhou** [comment]
ci:forward

**Comment by rotty3000** [comment]
Do we need to investigate something?

**Comment by shuyangzhou** [comment]
@vicnate5 please help me check the results, thanks!

**Comment by vicnate5** [comment]
@shuyangzhou Failures not related. Not sure why it re-ran after passing in https://github.com/shuyangzhou/liferay-portal/pull/10385#issuecomment-956933660

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/109435

**Comment by Kvale1733** [comment]
No regressions found during PR review. No additional testing required.

**Pull request closed without merging**